<launch>
    <arg name="jpg_topic" default="/rgb/image_raw/compressed"/>
    <arg name="depth_topic" default="/depth_to_rgb/image_raw"/>
    <arg name="lidar_topic" default="/PavoScanNode/scan"/>
    <arg name="calibration_path" default="$(find mapping_robot)/calibration.txt"/>
    <arg name="lidar_bash" default="/home/stereye/pavo_ws/devel/setup.bash"/>
    <arg name="lidar_remote_ip" default="192.168.10.2"/>
    <arg name="lidar_host_ip" default="192.168.10.1"/>
    <arg name="camera_bash" default="/home/stereye/azure_ws/devel/setup.bash"/>
    <arg name="worker_bash" default="$(find mapping_robot)/../../devel/setup.bash"/>
    <arg name="recorder_path" default="/home/stereye/data/last_record"/>
    <arg name="export_path" default="/home/stereye/data/scans"/>
    

    <node pkg="mapping_robot" type="gui_node" name="gui_node" output="screen" required="true">
        <param name="jpg_topic"                 type="string"   value="$(arg jpg_topic)" />
        <param name="depth_topic"           type="string"   value="$(arg depth_topic)" />
        <param name="calibration_path"  type="string"   value="$(arg calibration_path)"/>
        <param name="lidar_topic"              type="string"   value="$(arg lidar_topic)" />
        <param name="lidar_bash"              type="string"    value="$(arg lidar_bash)"/>
        <param name="lidar_remote_ip"  type="string"    value="$(arg lidar_remote_ip)"/>
        <param name="lidar_host_ip"        type="string"    value="$(arg lidar_host_ip)"/>
        <param name="camera_bash"       type="string"    value="$(arg camera_bash)"/>
        <param name="worker_bash"        type="string"    value="$(arg worker_bash)"/>
        <param name="recorder_path"     type="string"    value="$(arg recorder_path)"/>
        <param name="export_path"         type="string"    value="$(arg export_path)"/>
    </node>
</launch>
